.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,85 @@
+name: Deploy
+on:
+  push:
+    branches-ignore:
+      - 'main'
+    paths:
+      - 'app/**'
+      - 'docs/**'
+      - '.github/workflows/deploy*.yml'
+      - 'package.json'
+      - '.vercelignore'
+jobs:
+  deploy-preview:
+    name: Preview
+    runs-on: ubuntu-latest
+    steps:
+      - uses: chrnorm/deployment-action@v1.2.0
+        name: Create GitHub deployment
+        id: deployment
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          environment: Preview
+
+      - uses: actions/checkout@v2
+      # - name: Setup Ruby
+      #   uses: actions/setup-ruby@v1
+      #   with:
+      #     ruby-version: 2.7.x
```

Comment:
should we delete comments in the `yml` files?

Reply:
I'm uncommenting them in https://github.com/primer/view_components/pull/996, left them here as a sign they will be used imminently, and to leave more of a paper trail 🔍